packages/models/files.js

```diff
@@ -7,6 +7,12 @@ const Files = new FilesCollection({
     debug: true,
     collectionName: 'files',
     protected(fileObj) {
+        var session = this.params.query.xmtok;
+        var userId  = (Meteor.server.sessions[session] && Meteor.server.sessions[session].userId) ? Meteor.server.sessions[session].userId : null;
```

Comment:
@thomasspiesser @menelike 
Found it! `var userId = Meteor.server.sessions[token].userId`. No need to poll DB